ingest-app/src/cmr/ingest/api/provider.clj

```diff
@@ -43,6 +44,26 @@
      :headers {"Content-Type" (mt/with-utf-8 mt/json)}
      :body body}))
 
+(defn- result->response-map-sans-item
+  "Transforms an API result into a response map, removing specified items from the body.
+
+  Parameters:
+  - result: A map containing :status and :body keys, where :body is a JSON string.
+  - remove-list: A list of keys to be removed from the parsed JSON body.
+
+  Returns:
+  A map with :status, :headers, and :body keys, where:
+  - :status is the original status code
+  - :headers specify the Content-Type as JSON with UTF-8 encoding
+  - :body is the parsed JSON with specified keys removed"
+  [result remove-list]
+  (let [{:keys [status body]} result
+        metadata (-> body
+                     (json/parse-string))]
```

Comment:
you can pass true to avoid using string keys

`(json/parse-string true)  // Returns {:metadata ...}`